### DISCUSSION TITLE
Welcome to BenchmarkDotNet Discussions!

### DISCUSSION THREAD

**Discussion by AndreyAkinshin** [dotnet/BenchmarkDotNet#2331] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KSemenenko** [comment]
Is there any properly way to compare database performance?
For example, I run MySQL and postgreesql in the docker and measure the requests from the .net client.

In this case, do I need to warm up the cache or not? Many questions :)
